core/loaders/json.ts

```diff
@@ -1,11 +1,12 @@
+import { parse } from "../../deps/jsonc.ts";
 import { isPlainObject, read } from "../utils.ts";
 
 import type { Data } from "../../core.ts";
 
-/** Load and parse a JSON file */
+/** Load and parse a JSON / JSONC file */
 export default async function json(path: string): Promise<Data> {
   const text = await read(path, false);
-  const content = JSON.parse(text);
+  const content = parse(text);
```

Comment:
For performance reasons, I'd use jsonc only for files with the jsonc extension:

```js
const content = path.endsWith(".jsonc") ? parse(text) : JSON.parse(text);
```

Reply:
> For performance reasons, I'd use jsonc only for files with the jsonc extension:

OK, it has been updated.